.gitignore

```diff
@@ -51,3 +51,4 @@ mlrun_bc_*.json
 
 # used for local development.
 playground
+patch-env.yml
```

Comment:
`automation/patch_igz/patch_env.yml` is in line 41

Reply:
what I wrote is patch-env.yml with '-' in the middle instead of existing ignore with '_'.
Currently the file is as I described, but I do not know if it had another name before that, so let's keep them both.